SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -335,7 +299,8 @@ private void SetInstanceCustomProperties(CustomProperties customProperties)
         {
             if (!Configured)
             {
-                AppCenterLog.Error(AppCenterLog.LogTag, NotConfiguredMessage);
+                AppCenterLog.Error(AppCenterLog.LogTag, "App Center hasn't been configured. " +
```

Comment:
If resharper told you to inline this as only 1 occurrence, we should not follow the advice as we may add new calls in the future.

Reply:
@guperrot This is just revert. My change was vice versa :grin:

Reply:
Can you restore the constant in the message? It was nice change.